tests/lib/testlib/constants.py

```diff
@@ -0,0 +1,54 @@
+#
```

Comment:
If I am not wrong, the idea is to store all the tests constants in this file, right? If so, I think we are missing some constants but we could refactor them later when modifying the tests. Is too much work and unrelated to this PR so I am just adding this as a reminder.

Reply:
I created this file just to save the duplication of newly added test case to keep this PR small.

I don't want to touch other test cases.